labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -61,7 +76,12 @@ object RecursionPatternMatchingExercise {
    * List(1,1,2,3,1,1) -> List((4,1),(1,2),(1,3))
    */
   def amountEqualMembers[T](in: List[T]): List[(Int, T)] = {
-    error("fix me")
+     in match {
+      case Nil => Nil
+      case x :: xs => {
```

Comment:
I know a lot of examples use x and xs, and maybe that's more Scala-y and I just don't know it, but I think it would be nice to use something more descriptive than "x"